src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  const GROUP_STYLE = sourceString.trim().split(';');
```

Comment:
Using the variable name 'GROUP_STYLE' does not clearly convey the intent of the variable. Consider using a more descriptive name such as 'styleDeclarations' to improve readability.